docs/create/runtime-api.md

```diff
@@ -0,0 +1,137 @@
+---
+id: runtime-api
+title: Runtime API
+---
+
+The Runtime API exposes everything you could possibly want to access within the Webaverse runtime. This includes access to crypto minting, the world scene, camera, physics, and more.
+
+You can call these directly within your app without any extra requirements on your end. This is because the Runtime API is exposed to your code at runtime.
+
+## `renderer`
+This provides a THREE.js renderer at runtime that can be used to `setAnimationLoop` on.
+
+## `scene`
+This provides a THREE.js scene at runtime.
+
+## `camera`
+This provides a THREE.js camera at runtime.
+
+## `crypto`
+
+### `mintToken`
+
+#### Parameters
+
+| Name| Type | Description |
+| --- | ---- | ----------- |
+| `file` | blob | The data of the file to be tokenized |
+| `options` | object | You can define the `description` of the token here |
+
+#### Returns
+| Name| Type | Description |
+| --- | ---- | ----------- |
+| `tokenId` | number | The id of the newly minted token |
+
+-------------------
+
+## `physics`
+
+### `addGeometry`
+
+#### Parameters
+
+| Name| Type |
+| --- | ---- | 
+| `mesh` | mesh |
+
+-------------------
+
+### `removeGeometry`
+
+#### Parameters
+
+| Name| Type |
+| --- | ---- | 
+| `physicsId` | number |
+
+-------------------
+
+### `getRigTransforms`
+
+#### Returns
+```js
+[
+  {
+    position: this.localRig.inputs.leftGamepad.position,
+    quaternion: this.localRig.inputs.leftGamepad.quaternion,
+  },
+  {
+    position: this.localRig.inputs.rightGamepad.position,
+    quaternion: this.localRig.inputs.rightGamepad.quaternion,
+  },
+];
+```
+
+-------------------
+
+
+## `app`
+
+### `object.add`
+
+#### Parameters
+
+| Name| Type |
+| --- | ---- | 
+| `mesh` | mesh |
+
+-------------------
+
+### `files[]`
+
+This is an array of files to load from relative paths.
+
+-------------------
+
+### `unload`
+
+This is an event that gets emitted when the app instance is deleted.
```

Comment:
The user should clean up anything their app is doing here, such as removing physics objects, timers, etc.

The `app.object` will be automatically removed from the scene so there is no need to clean up that.

Reply:
454f960